tests/integration/api/controllers/contact.spec.js

```diff
@@ -5,18 +5,31 @@ const userFactory = require('@factories/cht/users/users');
 const {expect} = require('chai');
 
 describe('Contact API', () => {
-  // just a random string to be added to every doc so that it can be used to retrieve all the docs
-  const commonWord = 'freetext';
-  const contact0 = utils.deepFreeze(personFactory.build({ name: 'contact0', role: 'chw', notes: commonWord }));
+  // NOTE: this is a common word added to contacts to fetch them
```

Comment:
praise: thank you for the context here!  It will be a lot easier to maintain these tests going forwards with this included!